src/workloads/scale/LLTMixed.yml

```diff
@@ -241,56 +241,81 @@ Actors:
         LogEvery: 15 minutes
         Blocking: None
 
-- Name: MixedBaseline.Low
+- Name: Low.Short.Insert
   Type: CrudActor
   Threads: *LowThreads
```

Comment:
One of the two Threads (line 246 and 253) is unused. I think it's the one within the phase, but not sure. 
I think. 
They would have to be the same, regardless.

Reply:
It looks like we have a bug here - whenever you place it, you get an error. Only when you have it in two places, it works. I didn't checked now, but it is what I faced before. 

And yes, they are supposed to be the same everywhere.

Reply:
Asking in slack. I'd point out that mixed_workloads_genny uses CrudActor and only sets it at the Actor level https://github.com/mongodb/genny/blob/ac2c06a1ec477c569ea27f818a549b38cad08e01/src/workloads/scale/MixedWorkloadsGenny.yml

Reply:
https://mongodb.slack.com/archives/C01VD0LQZED/p1620307395080700

Reply:
Raiden and Ryan agree that setting it within the phase doesn't do anything and TIG-2766 should catch such settings in the future. Can you pull out the threads from the phases, and just sanity check that it still runs?